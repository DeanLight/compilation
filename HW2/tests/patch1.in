5 43 obj <<>> endobj 4