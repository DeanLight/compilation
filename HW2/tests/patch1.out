Producing Obj
Producing Body
Producing Dict
Producing KVList
Finished producing KVList
Finished producing Dict
Finished producing Body
Finished producing Obj
Syntax error
